readme.md

```diff
@@ -168,72 +384,67 @@ var Hello = React.createClass({
   }
 });
 
-React.render(
+ReactDOM.render(
   <Hello name="Tony" age="21" />,
-  document.getElementById( "container" )
+  document.getElementById( "app" )
 )
 ```
 
-## Exercise: A Blog Post (15 / 45)
+> **NOTE:** The return statement in `render` can only return one DOM element. You can, however, place multiple elements within a parent DOM element, like we do in the previous example with `<div>`.
+
```

Comment:
when are we going to introduce the idea of nested components. Think of it as a tree like structure